src/util/response.js

```diff
@@ -0,0 +1,39 @@
+const logger = require('../lib/logger/bunyan');
```

Comment:
Hey @whyDontI,

Will love to know how the logging levels are set in bunyan.

How we can use it while running the code

Reply:
I'll add the logResponse function in next the commit. I kind of took it for granted.
The logging level is 'info' so whatever above 'info' will be logged.